modules/sonarqube-service/main.tf

```diff
@@ -26,12 +26,13 @@ module "sonarqube-service" {
     load_balancer = "${var.loadbalancer_arn}"
   }
 
-  task_container_image              = "teliaoss/sonarqube-aws-env:7.3"
+  task_container_image              = "teliaoss/sonarqube-aws-env:1.0.3"
```

Comment:
This has to be `v1.0.3` since that is the tag you'll be using to tag the commit.